PrimeNumber.java

```diff
@@ -0,0 +1,23 @@
+import java.util.*;
+class PrimeNumber{
+	public static boolean isPrime(int n)
+	{
+		if (n <= 1)
+			return false;
+        
+        for (int i = 2; i <= Math.sqrt(n); i++)
```

Comment:
Add a comment before the for loop explaining the need of it

Reply:
ok